app/models/post.rb

```diff
@@ -22,6 +35,13 @@ def markdown_body
     render_as_markdown(body)
   end
 
+  def update_rank
+    age_in_days = (created_at - Time.new(1970,1,1)) / (60 * 60 *24)
```

Comment:
Space missing after comma.<br>Surrounding space missing for operator `*`.<br>Do not use `Time.new` without zone. Use one of `Time.zone.local`, `Time.new.current`, `Time.new.in_time_zone`, `Time.new.utc`, `Time.new.getlocal`, `Time.new.iso8601`, `Time.new.jisx0301`, `Time.new.rfc3339`, `Time.new.to_i`, `Time.new.to_f` instead.